.github/actions/setup-os/action.yaml

```diff
@@ -20,8 +20,8 @@ runs:
         # Install git-lfs to avoid compilation errors of "go" with Intel
         sudo apt-get install git-lfs
 
-        # Install krb5 to avoid compilation errors of "hdf" on Ubuntu
-        sudo apt-get install krb5-user libkrb5-dev
+        ## Install krb5 to avoid compilation errors of "hdf" on Ubuntu
```

Comment:
Note to self. Add a comment here that installing krb via apt leads to duplicate packages being built. Do after CI tests ran.